runner.sh

```diff
@@ -3,16 +3,11 @@
 # Copyright © 2023 Barış DEMİRCİ <hi@338.rocks>
 # SPDX-License-Identifier: GPL-3.0
 
-# This script is used to create docker image and run the container
-# It's main purpose is to download the latest ncfs.sh script from github
-# and run it inside the container
-# Nothing fancy. Just a simple script to get the latest ncfs.sh script
+# This script is used to run the ncfs.sh script which is packaged with the container
+# Simple script to execute the ncfs.sh in the container
 
-# Get the latest ncfs.sh script from github
-wget https://raw.githubusercontent.com/barbarbar338/ncfs/main/ncfs.sh
```

Comment:
Everything seems awesome but this part should be in the script. I don't want users to update their container every time when a new *basic* script update happens. This approach ensures people have the latest ncfs file wether they update their container or not because probably we are not going to update the docker image very frequently compared to ncfs script itself.